src/clj/commiteth/eth/multisig_wallet.clj

```diff
@@ -2,8 +2,23 @@
   (:require [commiteth.eth.core :as eth]
             [clojure.tools.logging :as log]))
 
+(def factory "0xbcBc5b8cE5c76Ed477433636926f76897401f838")
+
+(def factory-topic "0x96b5b9b8a7193304150caccf9b80d150675fa3d6af57761d8d8ef1d6f9a1a909")
+
```

Comment:
This topic hex-string looks a bit "magic". Perhaps add a comment explaining what it is?